### PR TITLE
fix: add extra security to file endpoints

### PR DESCRIPTION
**Pull request by rephus**
Closes:

### CODE REVIEW

File: packages/backend/src/routers/projectRouter.ts [hunk 1]
```diff
@@ -242,21 +242,20 @@ projectRouter.get(
...
-                throw new NotFoundError(
-                    `CSV file not found ${req.params.fileId}`,
-                );
+            const { fileId } = req.params;
+
+            if (!fileId.startsWith('csv-') || !fileId.endsWith('.csv')) {
+                throw new NotFoundError(`CSV file not found ${fileId}`);
             }
-            const filePath = path.join('/tmp', req.params.fileId);
+            const sanitizedFileId = fileId.replace('..', '');
```
Comment (IrakliJani) [inline packages/backend/src/routers/projectRouter.ts:250]: this is the most important change in both files right?

**Review by IrakliJani — APPROVED** [review]

File: packages/backend/src/routers/projectRouter.ts (hunk 1 above)
Comment (rephus) [reply packages/backend/src/routers/projectRouter.ts:250]: yep

Comment (IrakliJani) [reply packages/backend/src/routers/projectRouter.ts:250]: 🚢

**Comment by rephus** [comment]
I'll merge to fix this, if we want to use a different path method, we can do it on a separate PR

**Pull request merged**